### PR TITLE
Fix/M2-1875 Individual Schedule: calendar icon is not displaying from the left side of the respondent's name with the scheduled event

### PR DESCRIPTION
**Pull request by IhorNesterenko99**
…m the left side of the respondent's name with the scheduled event

### CODE REVIEW

File: src/modules/Dashboard/state/Users/Users.thunk.ts [hunk 1]
```diff
@@ -15,6 +15,17 @@ export const getWorkspaceRespondents = createAsyncThunk(
   },
 );
 
+export const getAllWorkspaceRespondents = createAsyncThunk(
```
Comment (kdanilenka32) [inline src/modules/Dashboard/state/Users/Users.thunk.ts:18]: Why need the thunk?

Comment (IhorNesterenko99) [reply src/modules/Dashboard/state/Users/Users.thunk.ts:18]: To divide respondents with the limit of 20 from all respondents. In the schedule I need to receive all respondents.

Comment (kdanilenka32) [reply src/modules/Dashboard/state/Users/Users.thunk.ts:18]: the do not have limit by default on back?
image

File: src/modules/Dashboard/features/Applet/Schedule/Legend/Legend.tsx [hunk 2]
```diff
@@ -34,7 +34,7 @@ export const Legend = ({ legendEvents, appletName, appletId }: LegendProps) => {
   const { t } = useTranslation('app');
   const { respondentId } = useParams();
   const navigate = useNavigate();
-  const { result: respondentsData } = users.useRespondentsData() || {};
+  const { result: respondentsData } = users.useAllRespondentsData() || {};
```
Comment (kdanilenka32) [inline src/modules/Dashboard/features/Applet/Schedule/Legend/Legend.tsx:37]: and in that case do y really need redux?

File: src/modules/Dashboard/state/Users/Users.thunk.ts (hunk 1 above)
Comment (IhorNesterenko99) [reply src/modules/Dashboard/state/Users/Users.thunk.ts:18]: thanks for clarifying, I thought there is no limit by default

Comment (IhorNesterenko99) [reply src/modules/Dashboard/state/Users/Users.thunk.ts:20]: added limit 10000

**Review by kdanilenka32 — APPROVED** [review]

**Pull request merged**